### ISSUE TITLE
No revenue

### ISSUE THREAD

**Issue by filippovpa** [hpool-dev/chia-miner#624]
My miner is shown online, but no plots shown
Trying new version, no good...
Some times reboot helps
But not this time
2 days = 0xch

**Comment by archerallstars** [comment]
The problem is due to these 2 factors:
1. Currently, at least for me, I have to use the miner version 1.4.1. Otherwise, my plots won't go online. I messed around with 1.4.0 but it's no use. Please upgrade to 1.4.1.
2. If I want to add plots to the miner, I will have to stop the miner in my machine first and wait until it goes offline (around 30 mins), then I also delete this offline miner on the web. After that, I can add plots to the miner and then start the miner again. It seems their backend is broken currently. If you add plots while your miner is still online, your signed plots will go down in number and they will be stuck there no matter how many plots you added later on.

**Comment by room101-dev** [comment]
I always notice in 'contribute' that the signed/un-signed ratio, drops usually about 50% seen this for months.
Normally what I do is run "chia plots check" thinking this could be related to the 'signed' problem, then of course I kill/re-start the miners.
I'm running multi-machine, and some miners just don't show up signed at all, and others just drop, almost always 3+ months when I start seeing this problem, I run 'check' and restart, then usually in a few hours the 'contribution' signed comes back to normal.
The bad is there is no notice, I just noticed today that yesterday at 8am I lost 50% of my 'signed' had been good for weeks.
Given that I have never deleted all miners 3+ months, it might be a good idea as mentioned here, To stop all wait until they can be deleted, and then re-run which rebuilds all their stuff from scratch.

The real problem seems to be this 'contribution', as online-miner doesn't mean anything, it just means your 'miner-client' offered N plots, the problem of course is that +20% of submitted plots to HPOOL are fake, and of course they must be super hard on all suspicious plots.

Lastly, this new update and mention to use 'xproxy' is very cryptic, I know they use xproxy for 'arweave', but its not clear how this can help this problem.

**Comment by room101-dev** [comment]
> My miner is shown online, but no plots shown
> Trying new version, no good...
> Some times reboot helps
> But not this time
> 2 days = 0xch

The problem is 'mining' seems to have no meaning, it just tells you that your miner has connected, of course what matters is 'online plot' and 'contribution'. The 'contribution' is troubling because its a bad chinese translation, better to use the 'chinese' and have google translate the real meaning.
So the problem is that 'miners' have no meaning that payment is based on 'contribution' and 'online-plot', where the plots must be 'signed', aka approved, now this is the problem, because we don't know which plots, we don't have any tools to know why plots are 'signed' ( approved ), we don't get any warning.
For instance if my plots go offline, I get an email, but if my plots online, aren't earning any compensation, I don't get any notice, so we end up having to check several times a day our 'contribution graph' to make sure we're getting paid for our 'online plots' 100%.

The problem seems to appear on miner's that are generating new plots, as my machines that just have HDD's with no new additions, don't seem to be having any of these payment problems.

The bad is once there is one 'bad file' the HPOOL seems to knock that entire system off the 'signed' list. It's a random problem, I have seen 3+ months, **_its troubling because if you don't check your payments several times a day, or your 'contribution plot', you have no idea your mining for ZERO xch._**

### **_HPOOL should have an email warning that tells us when our 'signed plots' is less than our 'submitted' ( mined plots )_**

**Comment by archerallstars** [comment]
> The problem seems to appear on miner's that are generating new plots, as my machines that just have HDD's with no new additions, don't seem to be having any of these payment problems.

Yes, I also don't have any problem with the current running miner. The problem arises as soon as I start adding new plots while my miner has an online status on the web. And strangely enough, my miner won't go offline for as long as 30 mins after I turned it off. Therefore it's a very time-consuming process when adding new plots to the pool. 

I don't use x-proxy. I think x-proxy is only necessary when you have many miners in the same network.
